Sources/MapboxCoreNavigation/SimulatedLocationManager.swift

```diff
@@ -232,7 +232,7 @@ open class SimulatedLocationManager: NavigationLocationManager {
            let nextCoordinateOnRoute = originalShape.coordinates.after(index: closestCoordinateOnRouteIndex),
            let time = expectedSegmentTravelTimes.optional[closestCoordinateOnRouteIndex] {
             let distance = originalShape.coordinates[closestCoordinateOnRouteIndex].distance(to: nextCoordinateOnRoute)
-            currentSpeed = min(max(distance / time, minimumSpeed), maximumSpeed)
+            currentSpeed = min(max(distance / max(time, 1.0), minimumSpeed), maximumSpeed)
```

Comment:
Could you please elaborate on why `time` cannot be less than 1 sec?

Reply:
time here is `expectedSegmentTravelTimes`. I chose 1 second as minimal time by which some segment could be traversed.